src/System.Xml.XDocument/src/System.Xml.XDocument.csproj

```diff
@@ -21,6 +21,9 @@
     <Compile Include="$(CommonPath)\System\SR.cs">
       <Link>Resources\SR.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\System\IO\StringBuilderCache.cs">
+      <Link>System\StringBuilderCache.cs</Link>
```

Comment:
orthogonal to this change but we should probably try to come up with a consistent way of referencing and setting the Link for the common files.